app/views/submissions/index.html.erb

```diff
@@ -16,6 +16,7 @@
 <% end %>
 
 <% content_for :javascripts do %>
+  <%= javascript_include_tag 'application' %>
```

Comment:
This is not needed. The layout imports application.js at the beginning of the `<body>` tag

Reply:
Issue addressed